libs/search.js

```diff
@@ -5,6 +5,26 @@ const get = require('lodash.get');
 const ELEMDELIM = '__';
 const MODDELIM = '_';
 
+function getFileName(path) {
```

Comment:
I suggest you use native nodejs [path](https://nodejs.org/api/path.html) module .